portfolio/src/main/java/com/google/sps/data/Post.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.data;
+
+public class Post {
+
+  private final String postId;
+  private long likeCount;
+  private long loveCount;
+  private long wowCount;
+  private long laughCount;
```

Comment:
I initially did use a Map, however maps aren't able to be stored in the datastore and so it requires manually unpacking the Map each time, or creating a map everytime I use this intermediate representation. It would reduce duplication but add some boilerplate (and some overhead) to extract the values out each time.

I shall change it to use a map, as the overhead is probably not very big, and it would be more flexible to future reactions.